test/client/bookmarkUtil.js

```diff
@@ -2,13 +2,14 @@ const test = require('tape')
 const bookmarkUtil = require('../../client/bookmarkUtil')
 
 test('test bookmarkUtil', (t) => {
-  t.plan(19)
+  t.plan(21)
   t.equals(bookmarkUtil.getBaseBookmarksOrder('0', 'ios'), '2.0.')
   t.equals(bookmarkUtil.getBaseBookmarksOrder('0', 'android'), '2.0.')
   t.equals(bookmarkUtil.getBaseBookmarksOrder('0', 'laptop'), '1.0.')
   t.equals(bookmarkUtil.getBaseBookmarksOrder('10', 'laptop'), '1.10.')
   t.equals(bookmarkUtil.getBookmarkOrder('', '2.0.9'), '2.0.8')
   t.equals(bookmarkUtil.getBookmarkOrder('2.0.8', ''), '2.0.9')
+  t.equals(bookmarkUtil.getBookmarkOrder('', '2.0.1'), '2.0.0.1')
```

Comment:
?
This doesn't test the change you made

Reply:
I had to re-do order calculation both in brave-core and brave-sync-js